editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -396,9 +397,22 @@ const NewCanvasControlsInner = (props: NewCanvasControlsInnerProps) => {
           if (frame == null) {
             return null
           }
-          const color = TP.isScenePath(path)
-            ? colorTheme.canvasSelectionSceneOutline.value
-            : colorTheme.canvasSelectionPrimaryOutline.value
+          const { components, imports } = getJSXComponentsAndImportsForPathInnerComponentFromState(
+            path,
+            props.editor,
+            props.derived,
+          )
+          const isFocusableComponent = MetadataUtils.isFocusableComponent(
+            path,
+            components,
+            componentMetadata,
+            imports,
+          )
+          const isFocusedComponent = TP.isFocused(props.editor.focusedElementPath, path)
+          const color =
+            isFocusableComponent || isFocusedComponent
```

Comment:
Is `isFocusedComponent` necessary? I would assume that only something focusable would be focused?